server/index.js

```diff
@@ -106,6 +106,8 @@ server.use("/", require("./routers/alpha"));
 server.use("/api", require("./api"));
 server.use("/admin", require("./routers/admin"));
 server.use("/assets", require("./routers/assets"));
+server.use("/createReminder", require("./routers/createReminder"));
```

Comment:
OK, but instead let's add a server/api/reminders_api.js that follows the API pattern established by other source files in that folder, for example, messages_api.js

POST /api/reminders          // create
GET /api/reminders/<id>   // retrieve
PUT /api/reminders/<id>   // update
DELETE /api/reminders/<id>  // delete